docs/manual_installation.md

```diff
@@ -0,0 +1,168 @@
+
+The second necessary configuration, is leveraging integrating the IPFS node, `go-ipfs` mentioned in the previous section, with Lotus. This configuration allows the IPFS node to be used as the underlying blockstore to store and retrieva data from the Filecoin network. This provides a convenient and efficient data-flow path in which Powergate won't participate, and thus avoid incurring in more overhead and resource usage.
+
+To integrate IPFS with the Lotus node, the following configuration settings should be set:
+- `Client.UseIpfs`/`LOTUS_CLIENT_USEIPFS`: With value `true`. This indicates that Lotus will use IPFS as the underlying blockstore for data for storage deals.
+- `Client.IpfsUseForRetrieval`/`LOTUS_CLIENT_IPFSUSEFORERETRIEVAL`: With value `true`. This indicates Lotus will use IPFS as the underlying blockstore for retrieval deals.
+- `Client.IpfsMAddr`/`LOTUS_CLIENT_IPFSMADDR`: Should contain the multiaddress of the IPFS node API mentioned in the last section, e.g: `/ip4/192.168.11.12/tcp/5001`.
+
+The above description wraps all necessary and minimal configuration to run Lotus for Powergate.
+
+### Wallet address keys
+
+As mentioned before, the default folder where Lotus will save all its state is `~/.lotus`. Apart from containing the `config.toml` file, it includes the `keystore` folder. This folder contains *all the keys of wallets created in Lotus*. It's essential to understand the consequences of this fact correctly. 
+
+The Lotus node will manage all wallets addresses used by Powergate, thus its private keys are contained in this folder. Therefore, take necessary precautions such as:
+- This folder should have minimal access.
+- Frequent backups are recommended for this folder. Recall that keys-backups are sensitive process to handle correctly since you're making copies of keys, and possibly increasing the risk of leakeage.
+- You can reduce the blast-radius of data loss by thinking about different strategies to keep the balances from these wallets addresses to a minimum.
+
+There's some discussion to allow Lotus to use an external component for signing, which allows to minimize these risks. Whenever that feature is ready, there's a high chance that this document will be updated to explain how to leverage that new feature for Powergate.
+
+
+# Install Powergate
+
+To run Powergate, you should run the `powd` binary with proper flags or environment variables configuration.
+
+You can download `powd` binaries from the [GitHub Releases section](https://github.com/textileio/powergate/releases) for your OS and architecture. If you prefer to compile `powd` from source, you can checkout the Powergate repo to your desired version and run `make build-powd` which will install `powd` in your `$GOPATH/bin` folder. Note that to compile, we're targeting Go 1.14 or newer version.
```

Comment:
Just remembered yesterday, this isn't actually true! We only provide pre-compiled `pow`. We should provide `powd` as well. Out of curiosity, I tried building it using `gox` and was getting some error for some Linux and Windows builds (maybe because I was trying to do this on my Mac?), so some work to do there I guess.

Reply:
Oh yeah, we should include `powd` in the releases. So I'll leave this because that will be true ~soon.

I'm pretty sure that error for Windows and other platforms is related to our fix some time ago to make `pow` compile for Windows. That's basically because a Lotus dependency wasn't wide-os compatible. I think that was merged in the original repo, but not sure yet if was merged downstream in Lotus deps.  Just a guess.